src/solc_0.7/catalyst/GemsCatalystsRegistry.sol

```diff
@@ -135,8 +136,7 @@ contract GemsCatalystsRegistry is WithSuperOperators {
         uint16 catalystId,
         uint256 amount
     ) public {
-        // @review what about metaTxs ?
-        require(msg.sender == from || isSuperOperator(msg.sender), "NOT_AUTHORIZED");
+        _checkAuthorization(from);
```

Comment:
Not much required to add metaTx access to these functions.
That said, you mentioned that the primary reason for this contract is to make life easier for AssetUpgrader and AssetMinter, so doesn't really make sense to enable metaTxs for everything.